oca/image.py

```diff
@@ -145,6 +145,16 @@ def chown(self, uid, gid):
         '''
         self.client.call(self.METHODS['chown'], self.id, uid, gid)
 
+    def clone(self, name='', ID=-1):
```

Comment:
Please replace `ID` with `id_`. It's a python convention to append an underscore. Upper case argument looks kind of weird :)